can-queues-test.js

```diff
@@ -158,11 +158,15 @@ if ( System.env.indexOf( 'production' ) < 0 ) {
 
 		canDev.log = function ( area, name ) {
 			QUnit.equal( "Test enqueuing:", area );
-			QUnit.equal( "fnName", name );
+			if (name) {
```

Comment:
I thought we would just skip tests that were about debugging features. Looks fine though.